### ISSUE TITLE
StagemonitorJdbcEventListener causes crash on flyway/springboot startup (on one-word statement)

### ISSUE THREAD

**Issue by jessegit** [stagemonitor/stagemonitor#314]
Relevant  part of exception,

```
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: -1
        at java.lang.String.substring(String.java:1967) ~[na:1.8.0_141]
        at org.stagemonitor.jdbc.StagemonitorJdbcEventListener.onAfterAnyExecute(StagemonitorJdbcEventListener.java:149) ~[stagemonitor-jdbc-0.82.0.jar!/:0.82.0]
        at com.p6spy.engine.event.SimpleJdbcEventListener.onAfterExecute(SimpleJdbcEventListener.java:113) ~[p6spy-3.0.0.jar!/:na]
```

The executed statement was "commit" without a space.

**Comment by felixbarny** [comment]
whoops. thx for reporting

**Comment by felixbarny** [comment]
I have just released `0.83.1`
